doozer/doozerlib/cli/release_gen_assembly.py

```
@@ -205,8 +207,10 @@ def __init__(
         suggestions_url: Optional[str] = None,
         gen_microshift: bool = False,
         release_date: Optional[str] = None,
+        registry_config: str = None,
```

Ruff (0.15.9)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)